src/components/Sticky/Sticky.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { STICKY } from '../ThemeIdentifiers';
+import { classNames } from '@shopify/react-utilities/styles';
+import * as baseTheme from './Sticky.scss';
+
+export type Position = 'top' | 'bottom';
+
+export interface Props {
+  children?: React.ReactNode;
+  position: Position;
+  style?: React.CSSProperties;
+  // Theme to be injected via css-themr.
+  theme?: any;
+}
+
+class Sticky extends React.PureComponent<Props, any> {
```

Comment:
@HiirenP since it's a stateless component, don't use class check other component like description or button

Reply:
resolve